web/shapeworks/src/views/Main.vue

```diff
@@ -302,17 +339,18 @@ export default {
             renderMetaData.value = newRenderMetaData
         }
 
-        const debouncedRefreshRender = _.debounce(refreshRender, 300)
+        const debouncedRefreshRender = _.debounce(refreshRender, 1000)
```

Comment:
Does this need to be extended for a particular reason?

Reply:
Thank you for pointing this out. I had originally done this to reduce redundant update calls before I had implemented the patchwise widget implementation. The render was getting refreshed too often as a user moved around the widgets. Now that we have patchwise updates, I can bring this number back down. I changed it back in [45ec6bc](https://github.com/girder/shapeworks-cloud/pull/355/commits/45ec6bcdae75c487dccf7b76d9e08a6c3e312a4c).